templates/pioneer-study/online-news/log/log.1.parquetmr.txt

```diff
@@ -0,0 +1,18 @@
+message pioneer-study-online-news-log {
+  required group metadata {
+    required int64  Timestamp;
+    required binary documentId (UTF8);
+    required binary pioneerId (UTF8);
+    required binary studyName (UTF8);
+    optional binary geoCity (UTF8);
+    optional binary geoCountry (UTF8);
+    optional binary normalizedChannel (UTF8);
+  }
+  required group entries (MAP) {
```

Comment:
Invalid parquet MAP schema and it looks like you want a LIST instead see: https://github.com/apache/parquet-format/blob/master/LogicalTypes.md